pkg/pprof/pprof.go

```diff
@@ -356,7 +356,6 @@ func (c *Converter) addExecutableInfo(
 		if !(os.IsNotExist(err) || errors.Is(err, fs.ErrNotExist)) {
 			// We don't want to log the error if the file doesn't exist,
 			// because it's probably a very short-lived process that weren't fast enough to obtain the FDs
-			level.Debug(c.logger).Log("msg", "failed to get executable info", "address", fmt.Sprintf("%x", addr), "err", err)
 		} else {
 			level.Warn(c.logger).Log("msg", "failed to get executable info", "address", fmt.Sprintf("%x", addr), "err", err)
```

Comment:
We also need to add one more check (perhaps in another PR), for `no such file or directory`, in case the file has been deleted

Reply:
Just realised that the current check is not very good... Just changed it